src/page-modules/assistant/trip-pattern/index.tsx

```diff
@@ -0,0 +1,93 @@
+import { formatLocaleTime, secondsToDuration } from '@atb/utils/date';
+import { type TripPattern } from '@atb/page-modules/assistant/server/journey-planner/validators';
+import style from './trip-pattern.module.css';
+import { ComponentText, useTranslation } from '@atb/translations';
+import { Typo } from '@atb/components/typography';
+import { motion } from 'framer-motion';
+import { Button } from '@atb/components/button';
+import { MonoIcon } from '@atb/components/icon';
+import { TransportIconWithLabel } from '@atb/components/transport-mode/transport-icon';
+
+type TripPatternProps = {
+  tripPattern: TripPattern;
+  index: number;
+};
+export function TripPattern({ tripPattern, index }: TripPatternProps) {
+  const { t, language } = useTranslation();
+
+  const duration = secondsToDuration(tripPattern.duration, language);
+  const fromPlace = tripPattern.legs[0].fromPlace.name ?? '';
+
+  return (
+    <motion.div
```

Comment:
I think actually this should be an anchor tag that wraps the entire element here. In modern HTML using `div` inside `a` is OK as `a` has a transparent content model (so everything but interactive elements essentially). And this should be links instead of buttons as we do a navigation. This would also make the entire element focusable / clickable

Reply:
Will implement that. 👍